src/specs.md

```diff
@@ -13,6 +13,41 @@ GNU Affero General Public License for more details.
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+```act
+behaviour wards of MkrAuthority
+interface wards(address usr)
+
+for all
+
+  May : uint256
+
+storage
+
+  wards[usr] |-> May
+
+iff
+
+  VCallValue == 0
+
+returns May
+```
+
+```act
+behaviour root of MkrAuthority
+interface root()
+
+types
+  Root : address
+
+storage
+  root |-> Root
```

Comment:
Since you added a storage alias for `root`, let's change the other proofs to also use it (right now they hardcode `0`).

Reply:
yup, they're all `root` now!